Library/Homebrew/cmd/shellenv.sh

```diff
@@ -41,8 +41,8 @@ homebrew-shellenv() {
       echo "setenv HOMEBREW_CELLAR ${HOMEBREW_CELLAR};"
       echo "setenv HOMEBREW_REPOSITORY ${HOMEBREW_REPOSITORY};"
       echo "setenv PATH ${HOMEBREW_PREFIX}/bin:${HOMEBREW_PREFIX}/sbin:\$PATH;"
-      echo "setenv MANPATH ${HOMEBREW_PREFIX}/share/man\`[ \${?MANPATH} == 1 ] && echo \":\${MANPATH}\"\`:;"
-      echo "setenv INFOPATH ${HOMEBREW_PREFIX}/share/info\`[ \${?INFOPATH} == 1 ] && echo \":\${INFOPATH}\"\`;"
+      echo "if ( \${?MANPATH} == 1 ) setenv MANPATH :\${MANPATH};"
```

Comment:
Unsure if there's an equivalent `set -e` handling needed for `fish`.

Reply:
Seeing how fish doesn't give any control over the separators, I don't think it's needed.

Reply:
@gromgit Sounds like from what @SMillerDev says this may not be needed for `fish` at all?

Reply:
It's more like some trickery might be necessary to get `fish` to Do The Right Thing. Let me do a few more experiments first.

Reply:
OK, figured out the necessary `fish` incantation.